File: 2714-left-and-right-sum-differences/left-and-right-sum-differences.cs
public class Solution {
    public int[] LeftRightDifference(int[] nums) {
        int total = 0;

        foreach (int x in nums) {
            total += x;
        }

        int left = 0;
        int[] ans = new int[nums.Length];

        for (int i = 0; i < nums.Length; i++) {
            total -= nums[i];
            ans[i] = Math.Abs(left - total);
            left += nums[i];
        }

        return ans;
    }
}

File: 2714-left-and-right-sum-differences/left-and-right-sum-differences.py
class Solution:
    def leftRightDifference(self, nums: List[int]) -> List[int]:
        total = sum(nums)
        left = 0
        ans = []

        for x in nums:
            total -= x
            ans.append(abs(left - total))
            left += x

        return ans
